src/main/scala/coreplex/SystemBus.scala

```diff
@@ -29,11 +28,9 @@ class SystemBus(params: SystemBusParams)(implicit p: Parameters) extends TLBusWr
 
 
   private val port_fixer = LazyModule(new TLFIFOFixer(TLFIFOFixer.all))
-  port_fixer.suggestName(s"${busName}_port_TLFIFOFixer")
```

Comment:
is the "busName" distintion no longer necesary due to hierarchy? Because I'd be annoyed if I end up with port_fixer and port_fixer_1 instead of these named port_fixers.

Reply:
These are now created as children of a module called busName already.

Reply:
and are as of #1190 created under further coupler submodules, e.g.: `busname.coupler_to_slave_namef_X.fixer`